demos-europe/demosplan-core
# chore: update EDT to version 0.12.21

Adjust dplan to the following changes:

* class renamings
* constructor parameter renamings
* constructor parameter removal



The new version also allows for better adjustments of `DrupalFilterValidator`, which is needed to properly continue with https://github.com/demos-europe/demosplan-core/pull/111.



### How to review/test

I don't think this needs an excessive review but here my guesses what you could do:

You could check the EDT changelog and verify that these changes are the only ones needed.

I've tested the segment list in EWM, you could do additional testing in EWM or other projects, especially in places that use the generic API.

You could also compare if the changes in the `services.yml` with the current class constructors.

### Linked PRs (optional)

* https://github.com/demos-europe/demosplan-documentation/pull/4

### Tasks (optional)


### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
